actions/move_milestones.go

```diff
@@ -0,0 +1,60 @@
+package actions
+
+import (
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/codegangsta/cli"
+	"github.com/deis/deisrel/git"
+	"github.com/google/go-github/github"
+)
+
+func MoveMilestone(ghClient *github.Client) func(c *cli.Context) []error {
```

Comment:
Can you make this a `func(c *cli.Context) error` - `(github.com/codegangsta/cli).Command`'s `Action` field is a `interface{}`, but we've standardized on `func(c *cli.Context) error`